python/ambassador/envoy/v2/v2listener.py

```diff
@@ -378,6 +378,8 @@ def v2filter_authv1(auth: IRAuth, v2config: 'V2Config'):
         }
 
     if auth_info:
+        auth_info['config']['clear_route_cache'] = True
```

Comment:
Hmmmm. Why is this needed now?

Reply:
clear_route_cache:
// Clears route cache in order to allow the external authorization service to correctly affect routing decisions. Filter clears all cached routes when: 
1. The field is set to *true*.
2. ...

More [here](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto#L63).

Reply:
Because we used to apply a patch to Envoythat always cleared the route cache, from before the 'clear_route_cache' flag landed in Envoy.

Reply:
@LukeShu's answer is what I needed. 🙂  Thanks!